schemas/core/components/common.json

```diff
@@ -73,6 +73,18 @@
       "type": "string",
       "minLength": 4
     },
+    "authToken": {
+      "description": "Authentication Token",
+      "type": "string",
+      "minLength": 2,
+      "maxLength": 255
+    },
+    "state": {
```

Comment:
Max length is too short. `authToken`, `state` and `nonce` are all query parameter string values. How about combining those under one type, e.g `encodedQueryParam`?